packages/block-library/src/post-comments/index.php

```diff
@@ -66,3 +66,20 @@ function register_block_core_post_comments() {
 	);
 }
 add_action( 'init', 'register_block_core_post_comments' );
+
+if ( ! function_exists( 'wp_post_comments_block_form_defaults' ) ) {
```

Comment:
We don't need this `function_exists` here because PHP functions declared in a `block-library/src/*/index.php` file will automatically be prefixed with `gutenberg_` when running in the Gutenberg plugin.

Reply:
Thanks! Done 👍